g2p/mappings/langs/str/str_equiv.json

```diff
@@ -1,74 +1,21 @@
 [
-    {
-        "in": "A\u0301",
-        "out": "Á"
-    },
-    {
-        "in": "C\u0301",
-        "out": "Ć"
-    },
-    {
-        "in": "I\u0301",
-        "out": "Í"
-    },
-    {
-        "in": "\u0335K",
-        "out": "₭"
-    },
-    {
-        "in": "Ḱ",
-        "out": "K\u0301"
-    },
-    {
-        "in": "Ḵ",
-        "out": "K\u0331"
-    },
-    {
-        "in": "K\u0332",
-        "out": "K\u0331"
-    },
-    {
-        "in": "\u0335L",
-        "out": "Ƚ"
-    },
-    {
-        "in": "Ṉ",
-        "out": "N\u0331"
-    },
-    {
-        "in": "N\u0332",
-        "out": "N\u0331"
-    },
-    {
-        "in": "S\u0301",
-        "out": "Ś"
-    },
-    {
-        "in": "T\u0331",
-        "out": "Ṯ"
-    },
-    {
-        "in": "T\u0332",
-        "out": "Ṯ"
-    },
-    {
-        "in": "\u0335T",
-        "out": "Ŧ"
-    },
-    {
-        "in": "W\u0332",
-        "out": "W\u0331"
-    },
-    {
-        "in": "X\u0332",
-        "out": "X\u0331"
-    },
-    {
-        "in": "¸",
-        "out": ","
-    },
-    {
-        "in": "¸",
-        "out": ","
-    }
-]
\ No newline at end of file
+    {"in": "Á", "out": "Á"},
+    {"in": "Ć", "out": "Ć"},
+    {"in": "Í", "out": "Í"},
+    {"in": "̵K", "out": "₭"},
+    {"in": "Ḱ", "out": "Ḱ"},
+    {"in": "Ḵ", "out": "Ḵ"},
+    {"in": "K̲", "out": "Ḵ"},
+    {"in": "̵L", "out": "Ƚ"},
```

Comment:
I prefer the escaped syntax here actually. I think `"̵L"` is less readable than `"\u0335L"`.  Similarly for the others, I prefer seeing e.g. `A\u0301` written out explicitly

Reply:
Meh, yeah, I wish we could easily have both visible. I find it a lot easier to glace at a file and see what's going on if I don't to remember the code points, and then put my cursor on them in VSCode to see which code point they are (with the Unicode Code Point plugin).
But I understand your point, especially for confusables, where \u0326 and \u0327 are visually nearly indistinguishable, yet distinct.
I don't mind reverting this rewrite, though, and keep the escape sequences that were there before I changed the formatting into the compact format.

Reply:
exactly, it's very hard to understand the difference between rules on line 7 and 8 otherwise, among others. Thanks for reverting it!

Reply:
Reverted. I'll merge this PR now.